pytest_pootle/fixtures/models/permission.py

```diff
@@ -54,3 +54,10 @@ def administrate(pootle_content_type):
     """Require the `suggest` permission."""
     return _require_permission('administrate', 'Can administrate a TP',
                                pootle_content_type)
+
+
+@pytest.fixture(scope="session")
+def translate(pootle_content_type):
```

Comment:
Though it is done in pytest_pootle.env I need `translate` permission`. I've preferred to create new fixture instead of getting it querying via ORM.